source/_templates/versioning.html

```diff
@@ -0,0 +1,34 @@
+{%- if versions %}
+{% if html_theme == 'sphinx_rtd_theme' %}
+<span class="rst-widget-header">
+    <span class="fa fa-book rst-widget-header-left"> Other Versions</span>
+    v: {{ current_version.version }}
```

Comment:
This sticks out a little because the Languages and Download selectors don't have similar text. Maybe remove `v: ` or remove it altogether? The version is already indicated at the top of the sidebar.